README.md

```diff
@@ -1 +1,38 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group_E_Simulation_Life_Cycle
```

Comment:
We could change it to "Group E: Tool for supporting the simulation lifecycle" or something like that

Reply:
I agree with Román that our project name should be more descriptive 😊 Since our program will manage different tools, data, processes... we could also consider something like this: "Group E: A solution for simulation lifecycle management".

Reply:
I like Françoise's suggestion

Reply:
This issue has been addressed. I'm resolving the conversation